### ISSUE TITLE
Failed building wheel for en-coref-sm

### ISSUE THREAD

**Issue by piyushk95** [huggingface/neuralcoref#42]
creating build\temp.win-amd64-3.6\Release\en_coref_sm
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.14.26428\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -IC:\Users\Piyush\Anaconda3\include -IC:\Users\Piyush\AppData\Local\Temp\pip-req-build-hvs3wmbv\include -IC:\Users\Piyush\Anaconda3\lib\site-packages\numpy\core\include -IC:\Users\Piyush\Anaconda3\include -IC:\Users\Piyush\Anaconda3\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.14.26428\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.14.26428\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\cppwinrt" /EHsc /Tpen_coref_sm/neuralcoref.cpp /Fobuild\temp.win-amd64-3.6\Release\en_coref_sm/neuralcoref.obj
  neuralcoref.cpp
  c:\users\piyush\anaconda3\lib\site-packages\numpy\core\include\numpy\npy_1_7_deprecated_api.h(12) : Warning Msg: Using deprecated NumPy API, disable it by #defining NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
  en_coref_sm/neuralcoref.cpp(5842): warning C4244: '=': conversion from 'Py_ssize_t' to 'int', possible loss of data
  en_coref_sm/neuralcoref.cpp(6639): warning C4244: '=': conversion from 'double' to 'float', possible loss of data
  en_coref_sm/neuralcoref.cpp(9894): warning C4244: '=': conversion from 'Py_ssize_t' to 'int', possible loss of data
  en_coref_sm/neuralcoref.cpp(9969): warning C4244: '=': conversion from 'Py_ssize_t' to 'int', possible loss of data
  en_coref_sm/neuralcoref.cpp(16201): warning C4244: '=': conversion from 'Py_ssize_t' to 'int', possible loss of data
  en_coref_sm/neuralcoref.cpp(17362): warning C4244: 'argument': conversion from 'uint64_t' to 'int', possible loss of data
  en_coref_sm/neuralcoref.cpp(17617): warning C4244: '=': conversion from 'int' to 'float', possible loss of data
  en_coref_sm/neuralcoref.cpp(17629): warning C4244: '=': conversion from 'int' to 'float', possible loss of data
  en_coref_sm/neuralcoref.cpp(17641): warning C4244: '=': conversion from 'int' to 'float', possible loss of data
  en_coref_sm/neuralcoref.cpp(17677): warning C4244: '=': conversion from 'double' to 'float', possible loss of data
  en_coref_sm/neuralcoref.cpp(17686): warning C4244: 'argument': conversion from 'uint64_t' to 'int', possible loss of data
  en_coref_sm/neuralcoref.cpp(17713): warning C4244: '=': conversion from 'double' to 'float', possible loss of data
  en_coref_sm/neuralcoref.cpp(17725): warning C4244: '=': conversion from 'int' to 'float', possible loss of data
  en_coref_sm/neuralcoref.cpp(18068): warning C4244: '=': conversion from 'double' to 'float', possible loss of data
  en_coref_sm/neuralcoref.cpp(30044): warning C4244: 'argument': conversion from '__pyx_t_5spacy_8typedefs_attr_t' to 'int', possible loss of data
  en_coref_sm/neuralcoref.cpp(31442): warning C4244: '=': conversion from '__pyx_t_5spacy_8typedefs_attr_t' to 'int', possible loss of data
  en_coref_sm/neuralcoref.cpp(31469): warning C4244: '=': conversion from '__pyx_t_5spacy_8typedefs_attr_t' to 'int', possible loss of data
  en_coref_sm/neuralcoref.cpp(31496): warning C4244: '=': conversion from '__pyx_t_5spacy_8typedefs_attr_t' to 'int', possible loss of data
  en_coref_sm/neuralcoref.cpp(31547): warning C4244: '=': conversion from '__pyx_t_5spacy_8typedefs_attr_t' to 'int', possible loss of data
  en_coref_sm/neuralcoref.cpp(49943): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.14.26428\bin\HostX86\x64\link.exe /nologo /INCREMENTAL:NO /LTCG /DLL /MANIFEST:EMBED,ID=2 /MANIFESTUAC:NO /LIBPATH:C:\Users\Piyush\Anaconda3\libs /LIBPATH:C:\Users\Piyush\Anaconda3\PCbuild\amd64 "/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.14.26428\ATLMFC\lib\x64" "/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.14.26428\lib\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.17134.0\ucrt\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.17134.0\um\x64" /EXPORT:PyInit_en_coref_sm/neuralcoref build\temp.win-amd64-3.6\Release\en_coref_sm/neuralcoref.obj /OUT:build\lib.win-amd64-3.6\en_coref_sm/neuralcoref.cp36-win_amd64.pyd /IMPLIB:build\temp.win-amd64-3.6\Release\en_coref_sm\neuralcoref.cp36-win_amd64.lib
  LINK : error LNK2001: unresolved external symbol PyInit_en_coref_sm/neuralcoref
  build\temp.win-amd64-3.6\Release\en_coref_sm\neuralcoref.cp36-win_amd64.lib : fatal error LNK1120: 1 unresolved externals
  error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\VC\\Tools\\MSVC\\14.14.26428\\bin\\HostX86\\x64\\link.exe' failed with exit status 1120

  ----------------------------------------
  **### Failed building wheel for en-coref-sm**
  Running setup.py clean for en-coref-sm
Failed to build en-coref-sm
Installing collected packages: en-coref-sm
  Running setup.py install for en-coref-sm ... error
    Complete output from command C:\Users\Piyush\Anaconda3\python.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\Piyush\\AppData\\Local\\Temp\\pip-req-build-hvs3wmbv\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\Users\Piyush\AppData\Local\Temp\pip-record-1qodkes3\install-record.txt --single-version-externally-managed --compile:
    model_name en_coref_sm
    model_dir en_coref_sm\en_coref_sm-3.0.0
    running install
    running build
    running build_py
    creating build
    creating build\lib.win-amd64-3.6
    creating build\lib.win-amd64-3.6\en_coref_sm
    copying en_coref_sm\__init__.py -> build\lib.win-amd64-3.6\en_coref_sm
    creating build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0
    copying en_coref_sm\en_coref_sm-3.0.0\meta.json -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0
    copying en_coref_sm\en_coref_sm-3.0.0\tokenizer -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0
    creating build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\ner
    copying en_coref_sm\en_coref_sm-3.0.0\ner\cfg -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\ner
    copying en_coref_sm\en_coref_sm-3.0.0\ner\lower_model -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\ner
    copying en_coref_sm\en_coref_sm-3.0.0\ner\moves -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\ner
    copying en_coref_sm\en_coref_sm-3.0.0\ner\tok2vec_model -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\ner
    copying en_coref_sm\en_coref_sm-3.0.0\ner\upper_model -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\ner
    creating build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\neuralcoref
    copying en_coref_sm\en_coref_sm-3.0.0\neuralcoref\cfg -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\neuralcoref
    copying en_coref_sm\en_coref_sm-3.0.0\neuralcoref\pairs_model -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\neuralcoref
    copying en_coref_sm\en_coref_sm-3.0.0\neuralcoref\single_model -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\neuralcoref
    creating build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\neuralcoref\static_vectors
    copying en_coref_sm\en_coref_sm-3.0.0\neuralcoref\static_vectors\key2row -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\neuralcoref\static_vectors
    copying en_coref_sm\en_coref_sm-3.0.0\neuralcoref\static_vectors\vectors -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\neuralcoref\static_vectors
    creating build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\neuralcoref\tuned_vectors
    copying en_coref_sm\en_coref_sm-3.0.0\neuralcoref\tuned_vectors\key2row -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\neuralcoref\tuned_vectors
    copying en_coref_sm\en_coref_sm-3.0.0\neuralcoref\tuned_vectors\vectors -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\neuralcoref\tuned_vectors
    creating build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\parser
    copying en_coref_sm\en_coref_sm-3.0.0\parser\cfg -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\parser
    copying en_coref_sm\en_coref_sm-3.0.0\parser\lower_model -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\parser
    copying en_coref_sm\en_coref_sm-3.0.0\parser\moves -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\parser
    copying en_coref_sm\en_coref_sm-3.0.0\parser\tok2vec_model -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\parser
    copying en_coref_sm\en_coref_sm-3.0.0\parser\upper_model -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\parser
    creating build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\tagger
    copying en_coref_sm\en_coref_sm-3.0.0\tagger\cfg -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\tagger
    copying en_coref_sm\en_coref_sm-3.0.0\tagger\model -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\tagger
    copying en_coref_sm\en_coref_sm-3.0.0\tagger\tag_map -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\tagger
    creating build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\vocab
    copying en_coref_sm\en_coref_sm-3.0.0\vocab\key2row -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\vocab
    copying en_coref_sm\en_coref_sm-3.0.0\vocab\lexemes.bin -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\vocab
    copying en_coref_sm\en_coref_sm-3.0.0\vocab\strings.json -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\vocab
    copying en_coref_sm\en_coref_sm-3.0.0\vocab\vectors -> build\lib.win-amd64-3.6\en_coref_sm\en_coref_sm-3.0.0\vocab
    copying en_coref_sm\meta.json -> build\lib.win-amd64-3.6\en_coref_sm
    running build_ext
    building 'en_coref_sm/neuralcoref' extension
    creating build\temp.win-amd64-3.6
    creating build\temp.win-amd64-3.6\Release
    creating build\temp.win-amd64-3.6\Release\en_coref_sm
    C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.14.26428\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -IC:\Users\Piyush\Anaconda3\include -IC:\Users\Piyush\AppData\Local\Temp\pip-req-build-hvs3wmbv\include -IC:\Users\Piyush\Anaconda3\lib\site-packages\numpy\core\include -IC:\Users\Piyush\Anaconda3\include -IC:\Users\Piyush\Anaconda3\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.14.26428\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.14.26428\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\cppwinrt" /EHsc /Tpen_coref_sm/neuralcoref.cpp /Fobuild\temp.win-amd64-3.6\Release\en_coref_sm/neuralcoref.obj
    neuralcoref.cpp
    c:\users\piyush\anaconda3\lib\site-packages\numpy\core\include\numpy\npy_1_7_deprecated_api.h(12) : Warning Msg: Using deprecated NumPy API, disable it by #defining NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
    en_coref_sm/neuralcoref.cpp(5842): warning C4244: '=': conversion from 'Py_ssize_t' to 'int', possible loss of data
    en_coref_sm/neuralcoref.cpp(6639): warning C4244: '=': conversion from 'double' to 'float', possible loss of data
    en_coref_sm/neuralcoref.cpp(9894): warning C4244: '=': conversion from 'Py_ssize_t' to 'int', possible loss of data
    en_coref_sm/neuralcoref.cpp(9969): warning C4244: '=': conversion from 'Py_ssize_t' to 'int', possible loss of data
    en_coref_sm/neuralcoref.cpp(16201): warning C4244: '=': conversion from 'Py_ssize_t' to 'int', possible loss of data
    en_coref_sm/neuralcoref.cpp(17362): warning C4244: 'argument': conversion from 'uint64_t' to 'int', possible loss of data
    en_coref_sm/neuralcoref.cpp(17617): warning C4244: '=': conversion from 'int' to 'float', possible loss of data
    en_coref_sm/neuralcoref.cpp(17629): warning C4244: '=': conversion from 'int' to 'float', possible loss of data
    en_coref_sm/neuralcoref.cpp(17641): warning C4244: '=': conversion from 'int' to 'float', possible loss of data
    en_coref_sm/neuralcoref.cpp(17677): warning C4244: '=': conversion from 'double' to 'float', possible loss of data
    en_coref_sm/neuralcoref.cpp(17686): warning C4244: 'argument': conversion from 'uint64_t' to 'int', possible loss of data
    en_coref_sm/neuralcoref.cpp(17713): warning C4244: '=': conversion from 'double' to 'float', possible loss of data
    en_coref_sm/neuralcoref.cpp(17725): warning C4244: '=': conversion from 'int' to 'float', possible loss of data
    en_coref_sm/neuralcoref.cpp(18068): warning C4244: '=': conversion from 'double' to 'float', possible loss of data
    en_coref_sm/neuralcoref.cpp(30044): warning C4244: 'argument': conversion from '__pyx_t_5spacy_8typedefs_attr_t' to 'int', possible loss of data
    en_coref_sm/neuralcoref.cpp(31442): warning C4244: '=': conversion from '__pyx_t_5spacy_8typedefs_attr_t' to 'int', possible loss of data
    en_coref_sm/neuralcoref.cpp(31469): warning C4244: '=': conversion from '__pyx_t_5spacy_8typedefs_attr_t' to 'int', possible loss of data
    en_coref_sm/neuralcoref.cpp(31496): warning C4244: '=': conversion from '__pyx_t_5spacy_8typedefs_attr_t' to 'int', possible loss of data
    en_coref_sm/neuralcoref.cpp(31547): warning C4244: '=': conversion from '__pyx_t_5spacy_8typedefs_attr_t' to 'int', possible loss of data
    en_coref_sm/neuralcoref.cpp(49943): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
    C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.14.26428\bin\HostX86\x64\link.exe /nologo /INCREMENTAL:NO /LTCG /DLL /MANIFEST:EMBED,ID=2 /MANIFESTUAC:NO /LIBPATH:C:\Users\Piyush\Anaconda3\libs /LIBPATH:C:\Users\Piyush\Anaconda3\PCbuild\amd64 "/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.14.26428\ATLMFC\lib\x64" "/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.14.26428\lib\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.17134.0\ucrt\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.17134.0\um\x64" /EXPORT:PyInit_en_coref_sm/neuralcoref build\temp.win-amd64-3.6\Release\en_coref_sm/neuralcoref.obj /OUT:build\lib.win-amd64-3.6\en_coref_sm/neuralcoref.cp36-win_amd64.pyd /IMPLIB:build\temp.win-amd64-3.6\Release\en_coref_sm\neuralcoref.cp36-win_amd64.lib
    LINK : error LNK2001: unresolved external symbol PyInit_en_coref_sm/neuralcoref
    build\temp.win-amd64-3.6\Release\en_coref_sm\neuralcoref.cp36-win_amd64.lib : fatal error LNK1120: 1 unresolved externals
    error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\VC\\Tools\\MSVC\\14.14.26428\\bin\\HostX86\\x64\\link.exe' failed with exit status 1120

    ----------------------------------------
Command "C:\Users\Piyush\Anaconda3\python.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\Piyush\\AppData\\Local\\Temp\\pip-req-build-hvs3wmbv\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\Users\Piyush\AppData\Local\Temp\pip-record-1qodkes3\install-record.txt --single-version-externally-managed --compile" failed with error code 1 in C:\Users\Piyush\AppData\Local\Temp\pip-req-build-hvs3wmbv\

**Comment by thomwolf** [comment]
Humm I think I see where it's coming from.
Let me just setup a Windows install to test the fix and I will update the models.

**Comment by thomwolf** [comment]
Hi @piyushk95, the problem should now be solved (it is working on my Windows install).

I have uploaded updated models. You just have to re-download the model and install it again.

The quickest way to install a model is `pip install MODEL_URL` in a clean environment and then use them with
```python
import en_coref_sm # or en_coref_md or en_coref_lg
nlp = en_coref_sm.load()
doc = nlp("I have a dog, he is very smart")
```

Please tell me if there is an issue.

**Comment by thomwolf** [comment]
Ok this should be fixed in the latest release (4.0).
